src/styles/trumps/_custom-element.scss

```diff
@@ -0,0 +1,3 @@
+.is-custom-element-disconnected {
```

Comment:
@LucaMele 
@pavestru 
@TheDadi 
We have two options:
- use a dedicated class, like above,
- or inline style like `<axa-foo style="visiblity: hidden;">`

**Important:** it has to be there before JS has downloaded (otherwise it wont work). So inlined would be the most bullet prove approach (and I think the really best).

Reply:
yes but style="visiblity: hidden;" has higher specificity which would require a `!important` selection or a `:root` dependency... so i dont know which is better.

What about the `hidden` attribute?

Reply:
every component that extends base component will remove its attribute hidden or similar logic

Reply:
@AndyOGo

Reply:
@LucaMele 
Thanks. Though the HTML `hidden` attribute is always overwritten by CSS `display`.
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden

Really weird that it is not the equivalent of CSS `visibility` 😕